compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -497,13 +501,69 @@ OMR::ARM64::TreeEvaluator::cloadEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    return commonLoadEvaluator(node, TR::InstOpCode::ldrhimm, cg);
    }
 
+// also handles vloadi
+TR::Register *
+OMR::ARM64::TreeEvaluator::vloadEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   return commonLoadEvaluator(node, TR::InstOpCode::vldrimmq, cg);
+   }
+
 TR::Register *
 OMR::ARM64::TreeEvaluator::awrtbarEvaluator(TR::Node *node, TR::CodeGenerator *cg)
 	{
 	// TODO:ARM64: Enable TR::TreeEvaluator::awrtbarEvaluator in compiler/aarch64/codegen/TreeEvaluatorTable.hpp when Implemented.
 	return OMR::ARM64::TreeEvaluator::unImpOpEvaluator(node, cg);
 	}
 
+TR::Register *
+OMR::ARM64::TreeEvaluator::inlineVectorBinaryOp(TR::Node *node, TR::CodeGenerator *cg, TR::InstOpCode::Mnemonic op)
```

Comment:
I don't think this function needs to be as a public member of `OMR::ARM64::TreeEvaluator`.
It can be a static function like `generateMaddOrMsub()` in BinaryEvaluator.cpp.